src/__tests__/Button.spec.js

```diff
@@ -7,4 +7,19 @@ describe('<Button />', () => {
 	it('renders without crashing', () => {
 		shallow(<Button />);
 	});
+
+	it('should render a button with component-button class', () => {
+		const wrapper = shallow(<Button/>);
+		expect(wrapper.find("component-button")).toBeTruthy();
+	});
+
+	it('should have name on props', () => {
+		const wrapper = shallow(<Button name="AC"/>);
+		expect(wrapper.instance().props.name).toBe('AC');
```

Comment:
Could have done something like:
`expect(wrapper.find('button').first().props().children).toBe('AC')` to see if it was rendering your prop correctly